### DISCUSSION TITLE
Tolerance test -- gaps missing?

### DISCUSSION THREAD

**Discussion by carlbeech** [supermerill/SuperSlicer#3624] [Q&A]
Hi

Got a (hopefully easy?) question...  new printer, so I'm still in the process of calibrating it... so I thought I'd download and do a tolerance test...  however checking I can see the small gaps have either been ignored, or superslicer has put lines in the gaps?   Can someone point me to the correct setting to keep the gaps in place?

![image](https://user-images.githubusercontent.com/31906429/216671095-3643c0eb-5b2f-4877-890e-6af766a8245a.png)

Superslicer 2.4

I thought it may be slice resolution or internal resolution - I've set these to 0.005 (rediculously small) but it didn't appear to have an effect...

Many thanks

Carl.

**Comment by carlbeech** [comment]
Ahhh - think I found it - Slice Gap closing radius - change from 0.049 to 0.01   :-)

Is there anything else I need to change?

**Reply by neophyl** [reply]
Yes that's the setting but you wouldn't change it from 0.049 to 0.01 as that closes up even bigger gaps so does the opposite of what you want.  If you want to keep the gaps from tolerance tests you would add more zeroes.  My normal is just to add 0.0049 as a gap has to be really small to meet that criteria.

**Reply by carlbeech** [reply]
Many thanks for the advice - much appreciated! :-)
